tests/unit/test_datafeeds.py

```diff
@@ -16,7 +16,7 @@
 """
 
 
-from collections import Callable
+from collections.abc import Callable
```

Comment:
This is for python 3.9 support?

Reply:
Just to get rid of the warning that's been bugging me for months. It will be required in 3.10 though.